Lib/test/test_typing.py

```diff
@@ -2792,6 +2829,10 @@ def test_forwardref_subclass_type_error(self):
         with self.assertRaises(TypeError):
             issubclass(int, fr)
 
+    def test_forwardref_only_str_arg(self):
```

Comment:
Should we put this one in 3.10 and 3.9 too?

Reply:
Yes, I agree. Will open a new one with just this change.